### PR TITLE
Update "ok to test" message.

### PR DESCRIPTION
**Pull request by spxtr**
I hope the transition won't be long, but people are confused so lets do this.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/520)

### CODE REVIEW

File: ciongke/cmd/hook/githubagent.go [hunk 1]
```diff
@@ -239,9 +239,11 @@ func (ga *GitHubAgent) trustedPullRequest(pr github.PullRequest) (bool, error) {
 
 func (ga *GitHubAgent) askToJoin(pr github.PullRequest) error {
 	commentTemplate := `
-Can a [%s](https://github.com/orgs/%s/people) member verify that this patch is reasonable to test? If so, please reply "ok to test".
+Can a [%s](https://github.com/orgs/%s/people) member verify that this patch is reasonable to test? If so, please reply with "ok to test" on its own line.
```
Comment (ixdy) [inline ciongke/cmd/hook/githubagent.go]: does it really need to be on its own line?

File: ciongke/cmd/hook/githubagent.go [hunk 2]
```diff
@@ -239,9 +239,11 @@ func (ga *GitHubAgent) trustedPullRequest(pr github.PullRequest) (bool, error) {
...
 
 Regular contributors should join the org to skip this step.
+
+While we transition away from the Jenkins GitHub PR Builder plugin, "ok to test" commenters will need to be on the admin list defined in [this file](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml).
```
Comment (rmmh) [inline ciongke/cmd/hook/githubagent.go]: Is the "While we..." clause necessary? Why not just say how it is, and we can change it later.

File: ciongke/cmd/hook/githubagent.go (hunk 1 above)
Comment (spxtr) [inline ciongke/cmd/hook/githubagent.go]: Yes, so that "this is not ok to test" doesn't trigger it.

File: ciongke/cmd/hook/githubagent.go (hunk 2 above)
Comment (spxtr) [inline ciongke/cmd/hook/githubagent.go]: During the transition, the jobs triggered by ciongke don't require this but the other jobs do. I want to make it clear that it's temporary.

**Pull request merged**